marc_to_solr/lib/princeton_marc.rb

```diff
@@ -641,10 +641,18 @@ def alma_code_start_53?(code)
   code.to_s.start_with?("53") && code.to_s.end_with?("06421")
 end
 
+def scsb_code_start?(code)
+  code.to_s.start_with?("scsb")
+end
+
 def alma_852(record)
   record.fields('852').select { |f| alma_code_start_22?(f['8']) }
 end
 
+def alma_and_scsb_852(record)
+  record.fields('852').select { |f| alma_code_start_22?(f['8']) || scsb_code_start?(f['b']) }
+end
```

Comment:
We have a check for scsb in princeton_marc, which is to check the value of the document id. SCSB starts with `SCSB-`, `scsb_doc?(record['001'].value)`. 
@mzelesky is it a robust check using subfield $b to check for `scsb` text? If yes then I think it's ok to have one more check besides the id.

Reply:
I think it's good to check the subfield b of an 852 for SCSB records. However, I think that the 'alma_and_scsb_852' needs to be separate checks. So, like we have 'alma_852', we should also have 'scab_852'. Then, if the record is identified as a SCSB record, go the scsb_852 route. If the record is identified as an All record, go the alma_852 route.

Reply:
@mzelesky is it possible to have a record that has both recap and non recap 852 fields?  I was unsure, which is why I put them together.